Formula/llvm.rb

```diff
@@ -17,7 +17,6 @@ class Llvm < Formula
     sha256 cellar: :any,                 big_sur:       "ff9a71b7b35ecb6c1dfcfe40152b00f4777a3f4a10dcf5cc41044458b02c99cd"
     sha256 cellar: :any,                 catalina:      "e9a8185649b019863424068c3efa1b3c7d85747d12260b1dac07895690f50383"
     sha256 cellar: :any,                 mojave:        "2382196cb9626c46aa27ce146270c6625deca1b8a47926457131fb22ba243899"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:  "ef3cc9c115ec2feb8ad2c317ddbfe190ad273a39acd737cd695fd700fb175c97"
```

Comment:
Please restore this line; test bot automation will update it for you.

Reply:
Done, and sorry. I did this based on the instructions in the old linuxbrew repository, I'll try to internalize the newer instructions.